src/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -52,10 +52,15 @@
     public static Integer mSortOrder = SORT_NAME;
     public static Boolean mSortAscending = true;
 
-    
+
     //private static final String LOG_TAG = "FileStorageUtils";
 
+    public static final boolean isStorageWritable() {
+        return Environment.getExternalStorageState() == Environment.MEDIA_MOUNTED;
```

Comment:
This should imho be implemented with an equals since you are comparing Strings and to make it null safe I would suggest:

``return Environment.MEDIA_MOUNTED.equals(Environment.getExternalStorageState());``